src/api/common/index.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
   dropsToXrp: utils.dropsToXrp,
   xrpToDrops: utils.xrpToDrops,
   toRippledAmount: utils.toRippledAmount,
-  generateWallet: utils.generateWallet,
+  generateAddress: utils.generateAddress,
```

Comment:
@vinniefalco I assume you're suggesting `generateAccountID`? I consider "address" a synonym for "accountID", so I think technically either works, but "address" is a bit easier to say and familiar from bitcoin.